### PR TITLE
Add ether support in currency class

### PR DESCRIPTION
**Pull request by medardm**
references #1

### CODE REVIEW

**Review by swengr-janan — APPROVED** [review]
Added!

File: src/entities/currency.ts [hunk 1]
```diff
@@ -17,6 +16,15 @@ export class Currency {
    * The only instance of the base class `Currency`.
    */
   public static readonly ETHER: Currency = new Currency(18, 'BNB', 'Binance')
+  public static readonly getEther = (chainId: ChainId) => {
+      switch (chainId) {
+          case 1:
+          case 5:
```
Comment (mativanarquero) [inline src/entities/currency.ts]: Replace with Goerli ETH instead of 'ETH' for chainid 5

File: src/entities/currency.ts [hunk 2]
```diff
@@ -17,6 +16,15 @@ export class Currency {
...
+              return new Currency(18, 'ETH', 'Ethereum')
```
Comment (mativanarquero) [inline src/entities/currency.ts:22]: Replace with Goerli ETH

**Review by mativanarquero — CHANGES_REQUESTED** [review]

**Review by acirrej — APPROVED** [review]

**Review by swengr-janan — APPROVED** [review]
AGITH!

**Pull request merged**